app/assets/javascripts/app/views/flash_messages_view.js

```diff
@@ -0,0 +1,18 @@
+app.views.FlashMessages = app.views.Base.extend({
+  templateName: "flash_messages",
+
+  flash: function(message, error){
+    this.presenter = {
+      message: message,
+      alertLevel: error ? "alert-danger" : "alert-success"
+    };
+
+    this.renderTemplate();
+    var flashElement = this.$el;
+
+    flashElement.addClass("expose");
+    setTimeout(function(){
+      flashElement.removeClass("expose");
+    }, 10100);
+  }
```

Comment:
I'd like to have two additional functions: `success` and `error` (which simply call `flash` with the correct values). Outside of this view only `success` and `error` should be used. You could also rename `flash` to `_flash`. That improves the readability of your code and makes this view easier to use.